### PR TITLE
Week 5 Assessment

### LINKED ISSUE #215: Weekly Assessment
Pull most often missed questions from wk1-4 assessments

### PR DESCRIPTION
**Pull request by memcmahon**

### CODE REVIEW

File: module3/assessments/week5-1422.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+layout: page
+title: Week 5 Assessment
+---
+
+[Follow the instructions on this repo!](https://github.com/turingschool-examples/Launch_6490)
```
Comment (rtillies) [inline module3/assessments/week5-1422.md]: This is the link to the Week 3 assessment repo

Comment (memcmahon) [reply module3/assessments/week5-1422.md]: uggghhh - numbers are hard.  It is updated with the correct link!

**Comment by zoefarrell** [comment]
I like the structure of this final assessment! Feels right to have full CRUD and also have 10/20 be passing. 

My one suggestion is I'm thinking might be better to remove the one-to-many entirely. I think the assessment should be focused on assessing CRUD for a single resource and the project should bring in one-to-many.

So maybe have nothing about clubs, the details page would show the same information as the main page, but I think that's fine. And the edit form is for editing the name or capacity, nothing with clubs.

What do you think?


Also, might be good to note how long you're planning to give students for this assessment. I could see doing the longer time again because it is a large ask. Maybe also worth an instructor note to clarify 10 points is passing and encourage students to focus on the routes they are most comfortable with first.

**Review by zoefarrell — APPROVED** [review]

**Comment by memcmahon** [comment]
In my experience with students this week and last, I think that they are really comfortable with CRUD-ing a single resource.  I really do want to push them to dig further, but they can pass if they don't get to (or can't make work) the one to many relationship. 

I do think we could make this assessment a little more 'do-able' if we remove the requirement for tests - each user story is worth 2 points, just for implementation.  Since we have not focused as much on TDD or asking students to build their own tests during lessons, I could see us dropping that requirement. (brownie points for students who write tests)

Definitely agree on the time expecation!

**Pull request merged**